kobuki_node/include/kobuki_node/diagnostics.hpp

```diff
@@ -17,16 +17,19 @@
 ** Includes
 *****************************************************************************/
 
+#include <vector>
+
 #include <kobuki_driver/packets/cliff.hpp>
 #include <kobuki_driver/modules/battery.hpp>
 #include <kobuki_driver/packets/core_sensors.hpp>
-#include <diagnostic_updater/diagnostic_updater.h>
+#include <diagnostic_updater/diagnostic_updater.hpp>
 
 /*****************************************************************************
 ** Namespaces
 *****************************************************************************/
 
-namespace kobuki {
+namespace kobuki_node
```

Comment:
Ah, just noticed this. Didn't remember it in the other PRs. We had originally namespaced everything under `kobuki` (like `ecl`) so you don't have the inconvenience of long-winded namespace calls for frequently used libraries and you don't have to worry about conflicts from other sources.

If you can unwind it easily, great. Otherwise I'll do a pass on all the libraries later.

Reply:
I've been usually going by the namespace of the package, which is why I changed it here.  I'll leave it for now and we can revisit this later.

Reply:
Ok